ui/app/models/pki/crl.js

```diff
@@ -4,15 +4,80 @@
  */
 
 import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
 
+@withFormFields(['expiry', 'autoRebuildGracePeriod', 'deltaRebuildInterval', 'ocspExpiry'])
 export default class PkiCrlModel extends Model {
   // This model uses the backend value as the model ID
   get useOpenAPI() {
     return true;
   }
+  getHelpUrl(backendPath) {
+    return `/v1/${backendPath}/config/crl?help=1`;
+  }
 
-  @attr('string') expiry;
   @attr('boolean') autoRebuild;
-  @attr('string') ocspExpiry;
-  @attr('boolean') ocspDisable;
+  @attr('string', {
+    label: 'Auto-rebuild on',
+    labelDisabled: 'Auto-rebuild off',
+    defaultValue: '12h',
+    editType: 'ttl',
+    mapToBoolean: 'autoRebuild',
+    helperTextEnabled: 'Vault will rebuild the CRL in the below grace period before expiration',
+    helperTextDisabled: 'Vault will not automatically rebuild the CRL',
+  })
+  autoRebuildGracePeriod;
+
+  @attr('boolean') enableDelta; // add validations auto_rebuild must be enabled
+  @attr('string', {
+    label: 'Delta CRL building on',
+    labelDisabled: 'Delta CRL building off',
+    defaultValue: '15mh',
```

Comment:
Is this supposed to be `15h` instead of `15mh` ?

Reply:
it should be 15m actually! good catch!